cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -739,6 +739,8 @@ spec:
             {{- if .Values.global.grafana }}
             - name: GRAFANA_ENABLED
               value: "{{ template "cost-analyzer.grafanaEnabled" . }}"
+            - name: LOG_LEVEL
+              value: {{ .Values.kubecostModel.logLevel }}
```

Comment:
@mittal-ishaan Is there a reason why this `LOG_LEVEL` env var is within an `if` statement here? We should probably be setting this outside the `if` statement.

Reply:
my bad here.  I should have been more careful. I will correct it.